static/app/routes.tsx

```diff
@@ -1885,9 +1885,14 @@ function routes() {
             <Route
               path="widget/new/"
               componentPromise={() =>
-                import(
-                  /* webpackChunkName: "WidgetNew" */ 'app/views/dashboardsV2/widget/new'
-                )
+                import(/* webpackChunkName: "Widget" */ 'app/views/dashboardsV2/widget')
+              }
+              component={errorHandler(LazyLoad)}
+            />
+            <Route
+              path="widget/:widgetId/edit/"
+              componentPromise={() =>
+                import(/* webpackChunkName: "Widget" */ 'app/views/dashboardsV2/widget')
```

Comment:
Someone left me a review on one of my PRs that `webpackChunkName` needs to be unique, is that the case here?